code/game/machinery/atmoalter/clamp.dm

```diff
@@ -0,0 +1,150 @@
+//Good luck. --BlueNexus
+
+//Static version of the clamp
+/obj/machinery/clamp
+	name = "stasis clamp"
+	desc = "A magnetic clamp which can halt the flow of gas in a pipe, via a localised stasis field."
```

Comment:
Extended desc should include how to remove it, if not by wrench as most other atmos-related things use